src/test/kotlin/com/pubnub/api/legacy/endpoints/presence/HereNowEndpointTest.kt

```diff
@@ -105,7 +105,7 @@ class HereNowEndpointTest : BaseTest() {
     @Test
     fun testMultipleChannelSync() {
         stubFor(
-            get(urlPathEqualTo("/v2/presence/sub_key/mySubscribeKey/channel/ch1,ch2"))
+            get(urlPathEqualTo("/v2/presence/sub-key/mySubscribeKey/channel/ch1,ch2"))
```

Comment:
Why this change?
In openApi there is "sub_key"

Reply:
I see PR from Serhii. I am wondering why openAPI doesn't reflect those changes?

Reply:
AFAIK our server doesn't care, and we have a mess on SDK side. Would be nice to have just one way of referencing it. This was chosen because at the time of change Serhii was in the office and on ObjectiveC everywhere is `-`. I don't care what character we will use, as long as it's consistent